katas/content/complex_arithmetic/complex_addition/Verification.qs

```diff
@@ -0,0 +1,54 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Random;    
+    open Microsoft.Quantum.Convert; 
+
+    operation ComplexAddExp(x : Complex, y : Complex) : Complex {
+    
+        // Calculate the sum of two complex numbers.  
+        let z = Complex (x::Real + y::Real, x::Imag + y::Imag);
+        return z;
+
+    }
+
+    operation ComplexRandom(min : Double, max : Double) : Complex{
+        
+        // Generates a random complex number. 
+        let real = DrawRandomDouble(min, max);
+        let imag = DrawRandomDouble(min, max);
+        return Complex (real, imag);
+        }
+
+    operation ComplexEqual(x : Complex, y : Complex) : Bool{
+
+        // Tests two complex numbers for equality.
+        return ((x::Real == y::Real) and (x::Imag == y::Imag));
```

Comment:
Real and Imag are floating-point fields so it's preferrable to compare them within some tolerance to account for floating point imprecision. You can find example in katas\content\qubit\learn_single_qubit_state\Verification.qs, we used floating point numbers there as well

Reply:
Revised